grievanceManagementSystemAPI/src/main/java/nucleusteq/com/grievance/entity/Comments.java

```diff
@@ -0,0 +1,131 @@
+package nucleusteq.com.grievance.entity;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+/**
+ * Represents a Ticket Comments entity for the grievance management system.
+ */
+@Entity
+@Table(name = "ticket_comments")
+public class Comments {
+
+  /**
+   * Comments Id.
+  */
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Integer commentId;
+
+  /**
+   * Comments.
+   */
+  @Column
+  private String comments;
+  /**
+   * Commented by.
+   */
+  private String commentedBy;
+
+  /**
+   * Commented Time.
+   */
+  private String time;
+  /**
+   * No args constructor.
+   */
+  public Comments() {
+  }
+
+  /**
+   * All arg constructor.
+   *
+   * @param commentsIdParam id.
+   * @param commentsParam comments.
+   */
+   public Comments(
+     final Integer commentsIdParam,
+     final String commentsParam) {
+    super();
+    this.commentId = commentsIdParam;
```

Comment:
It's not an all args constructor if all your args are not here right? Check this.